src/Processors/Formats/Impl/PrettyBlockOutputFormat.cpp

```diff
@@ -42,7 +42,7 @@ bool PrettyBlockOutputFormat::cutInTheMiddle(size_t row_num, size_t num_rows, si
 /// Evaluate the visible width of the values and column names.
```

Comment:
It looks like the variable `horizontal_bold_bar` [line 205](https://github.com/ClickHouse/ClickHouse/pull/74032/files#diff-b9c946c844ee8c90f75bab54aebecc2223032812634841a629913b0396469f3cL205) is not used anywhere? Not sure about CI but saw my build fail locally.